sympy/physics/continuum_mechanics/beam.py

```diff
@@ -209,6 +210,68 @@ def bc_deflection(self):
     def bc_deflection(self, d_bcs):
         self._boundary_conditions['deflection'] = d_bcs
 
+    def apply_support(self, loc, type="cantilever"):
+        """
+        This method applies support to a particular beam object.
+
+        Parameters
+        ==========
+        loc : Sympifyable
+            Location of point at which support is applied.
+        type : String
+            Determines type of Beam support applied.
+            - For Simple pinned Support, type = "pin"
+            - For Cantilever Support, type = "fixed"
+            - For Rolling Support, type = "roller"
```

Comment:
Cantilever is not a type of a support. I think this should be more specific about the degrees of freedom. Pin joint fixes 2 of 3 DoF, fixed 3 of 3 DoF, and roller, 1 of 3 DoF.